CDP4Dal.NetCore.Tests/MessageBusTestFixture.cs

```diff
@@ -126,31 +128,31 @@ public void VerifyMemoryUsage()
 
             var count = 26630;
             var repeat = 5;
-            var currentCount = CDPMessageBus.Current.ActiveObservableCount;
-            var currentCalls = CDPMessageBus.Current.ListenerCallCount;
+            var currentCount = this.messageBus.ActiveObservableCount;
+            var currentCalls = this.messageBus.ListenerCallCount;
 
             for (var i = 1; i <= count; i++)
             {
                 var obj = new Person() { ShortName = "User" };
 
                 for (var j = 0; j < repeat; j++)
                 {
-                    disposables.Add(CDPMessageBus.Current.Listen<ObjectChangedEvent>(obj).Where(x => true).Subscribe(x => { name = x.EventKind.ToString(); }));
-                    disposables2.Add(CDPMessageBus.Current.Listen<ObjectChangedEvent>(obj).Where(x => true).Subscribe(x => { name = x.EventKind.ToString(); }));
+                    disposables.Add(this.messageBus.Listen<ObjectChangedEvent>(obj).Where(x => true).Subscribe(x => { name = x.EventKind.ToString(); }));
+                    disposables2.Add(this.messageBus.Listen<ObjectChangedEvent>(obj).Where(x => true).Subscribe(x => { name = x.EventKind.ToString(); }));
                 }
             }
 
-            TestContext.WriteLine($"Count: {CDPMessageBus.Current.ActiveObservableCount}, Calls: {CDPMessageBus.Current.ListenerCallCount}");
+            TestContext.WriteLine($"Count: {this.messageBus.ActiveObservableCount}, Calls: {this.messageBus.ListenerCallCount}");
 
-            Assert.AreEqual(count + currentCount, CDPMessageBus.Current.ActiveObservableCount);
-            Assert.AreEqual((count * 2 * repeat) + currentCalls, CDPMessageBus.Current.ListenerCallCount);
+            Assert.AreEqual(count + currentCount, this.messageBus.ActiveObservableCount);
+            Assert.AreEqual((count * 2 * repeat) + currentCalls, this.messageBus.ListenerCallCount);
 
             GC.Collect();
 
             dotMemory.Check(x =>
             {
                 result2 = x.SizeInBytes;
-                Assert.IsTrue(result2 - result1 < 107000000, "CDPMessageBus uses more memory that expected on creation of Observables and Subscriptions");
+                Assert.IsTrue(result2 - result1 < 107000000, "SingletonCDPMessageBus uses more memory that expected on creation of Observables and Subscriptions");
```

Comment:
should this not be CDPMessageBus instead of SingletonCDPMessageBus? (present in the file couple of time)